apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/codegen/kotlin/KotlinCodeGenUtil.kt

```diff
@@ -0,0 +1,18 @@
+package com.apollographql.apollo.compiler.codegen.kotlin
+
+import com.squareup.kotlinpoet.AnnotationSpec
+import com.squareup.kotlinpoet.ClassName
+import com.squareup.kotlinpoet.FunSpec
+
+val MULTIPLATFORM_THROWS = ClassName.bestGuess("com.apollographql.apollo.api.internal.Throws")
```

Comment:
```suggestion
private val MULTIPLATFORM_THROWS = ClassName.bestGuess("com.apollographql.apollo.api.internal.Throws")
```

Reply:
I didn't see the constructor when writing this. 😊